### ISSUE TITLE
TownyChat - "Invisible Plugin" mode

### ISSUE THREAD

**Issue by LlmDl** [TownyAdvanced/Towny#944]
Originally reported on Google Code with ID 911

```
Would it be possible to make Towny Chat be "disabled" except for the /tc and /nc commands?
Thats the only thing I need it for, but even when the formatting is off it still interferes
with herochat and spoutcraft:

- Herochat, the formatting for console goes a bit iffy, making it harder to monitor
the channels via console (We have a lot of local channels) - also, in some channels
it just says "you feel lonely" when you talk even if people can hear you

- Spoutcraft, even if spam timer is set to 0.0, multiline chat is considered spam (with
spoutcraft you can type as much as you want, and its sent as a number of messages together)


TL;DR - any way to JUST use the /tc and /nc, without anything else at all from towny
chat?



```

Reported by `SilvaniaMCServer` on 2012-04-12 14:55:28
